src/annoymodule.cc

```diff
@@ -356,14 +359,15 @@ py_an_add_item(py_annoy *self, PyObject *args, PyObject* kwargs) {
 static PyObject *
 py_an_build(py_annoy *self, PyObject *args, PyObject *kwargs) {
   int q;
+  int n_threads;
   if (!self->ptr) 
     return NULL;
-  static char const * kwlist[] = {"n_trees", NULL};
-  if (!PyArg_ParseTupleAndKeywords(args, kwargs, "i", (char**)kwlist, &q))
+  static char const * kwlist[] = {"n_trees", "n_threads", NULL};
+  if (!PyArg_ParseTupleAndKeywords(args, kwargs, "i", (char**)kwlist, &q, &n_threads))
```

Comment:
replace `"i"` with `"i|i"` and set `int n_threads=1` as the default value. that way users don't have to provide a value (it will default to 1)

Reply:
I have ```if (!PyArg_ParseTupleAndKeywords(args, kwargs, "i|i", (char**)kwlist, &q, &n_threads))``` now is there another change needed for that line?

There should be a default for n_threads in annoylib.h, does that work?
```void build(int q, int n_threads = 1)```

Reply:
Fixed now (I think)